oxd-server/src/test/resources/testng.xml

```diff
@@ -247,6 +253,15 @@
             <class name="io.swagger.client.api.UmaGetClaimsGatheringUrlTest"/>
         </classes>
     </test>
-
-
+    <!-- Mock tests-->
```

Comment:
It should go in separate testng-load.xml.

Reply:
Done